test/testcafe/framework/page-objects/EnrollSecurityQuestionPageObject.js

```diff
@@ -10,14 +11,25 @@ export default class EnrollSecurityQuestionPageObject extends BasePageObject {
   }
 
   clickChooseSecurityQuestion() {
+    if (userVariables.v3) {
```

Comment:
Is there a better way to do this? i.e. not pass a dummy `` value as 1st argument.
In v3, we don't have a field name for radio options.
In v2, field name is used like this - this.el.find([data-se="${fieldName}"]`);`
In v3, we can use just the value field (2nd param) - `this.el.find(`input[type="radio"][value="${value}"]`);`

I don't want to create another method that just takes 1 argument, since methods with same names are not allowed according to lint rules.